data_model/src/events/time.rs

```diff
@@ -245,41 +245,35 @@ fn multiply_duration_by_u128(duration: Duration, n: u128) -> Duration {
 }
 
 impl Schedule {
-    /// Create new [`Schedule`] starting at `start` and without period
+    /// Create new [`Schedule`] starting at `start_ms` and without period
     #[must_use]
     #[inline]
-    pub const fn starting_at(start: Duration) -> Self {
+    pub const fn starting_at(start_ms: Duration) -> Self {
```

Comment:
```suggestion
    pub const fn starting_at(start: DateTime<Utc>) -> Self {
```

@mversic should this be a `u64` or a `DateTime`?